readthedocs/projects/models.py

```diff
@@ -112,7 +112,7 @@ def save(self, *args, **kwargs):
 
     # HACK
     def get_absolute_url(self):
-        return resolve(self.child)
+        return resolver.resolve_version(self.child)
```

Comment:
Would be useful to pass a kwarg here. I thought we were only using resolve_version when we had a version object?